test/e2e/e2e_ginkgo/e2e_ginkgo_suite_test.go

```diff
@@ -0,0 +1,81 @@
+package e2e_ginkgo
+
+import (
+	"flag"
+	"fmt"
+	"testing"
+
+	v1 "github.com/operator-framework/operator-lifecycle-manager/pkg/api/apis/operators/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+	"github.com/operator-framework/operator-lifecycle-manager/pkg/api/client"
+)
+
+
+var (
+	kubeConfigPath = flag.String(
+		"kubeconfig", "", "path to the kubeconfig file")
+
+	namespace = flag.String(
+		"namespace", "", "namespace where tests will run")
+
+	olmNamespace = flag.String(
+		"olmNamespace", "", "namespace where olm is running")
+
+	communityOperators = flag.String(
+		"communityOperators",
+		"quay.io/operator-framework/upstream-community-operators@sha256:098457dc5e0b6ca9599bd0e7a67809f8eca397907ca4d93597380511db478fec",
+		"reference to upstream-community-operators image")
+
+	dummyImage = flag.String(
+		"dummyImage",
+		"bitnami/nginx:latest",
+		"dummy image to treat as an operator in tests")
+
+	testNamespace           = ""
+	operatorNamespace       = ""
+	communityOperatorsImage = ""
+)
+
+
+func TestE2eGinkgo(t *testing.T) {
```

Comment:
nit: I would still have some comments before the functions.

Reply:
Added comments